includes/validation/class-amp-validated-url-post-type.php

```diff
@@ -1682,7 +1696,15 @@ public static function handle_validation_error_status_update() {
 				continue;
 			}
 			$term_group = AMP_Validation_Error_Taxonomy::sanitize_term_status( $status );
-			if ( null !== $term_group && $term_group !== $term->term_group ) {
+			if ( null === $term_group ) {
+				continue;
+			}
+
+			$term_group = $status_acknowledged
+				? $term_group | AMP_Validation_Error_Taxonomy::ACKNOWLEDGED_VALIDATION_ERROR_BIT_MASK
+				: $term_group - AMP_Validation_Error_Taxonomy::ACKNOWLEDGED_VALIDATION_ERROR_BIT_MASK;
```

Comment:
I assume this is meant to _disable_  the `ACK` flag?

This is done with bitmasks by an "and not" operation:
```suggestion
				: $term_group &  ~AMP_Validation_Error_Taxonomy::ACKNOWLEDGED_VALIDATION_ERROR_BIT_MASK;
```